yt/visualization/plot_container.py

```diff
@@ -943,7 +946,7 @@ def _sanitize_units(z, _field):
     def set_cbar_minorticks(self, field, state):
         """Deprecated alias, kept for backward compatibility.
 
-        turn colorbar minor ticks "on" or "off" in the current plot, according to *state*
+        turn colorbar minor ticks "on" or "off" in the current plot, following *state*
```

Comment:
I think the previous wording is clearer here.

Reply:
would this work ?
```suggestion
        turn colorbar minor ticks "on" or "off" in the current plot,
        according to *state*
```

Reply:
yep!